styles.css

```diff
@@ -8,61 +8,65 @@
   display: flex;
   flex-direction: column;
   align-items: center;
+  width: 375px;
 }
 
 .toolbar-container {
   display: flex;
   flex-direction: row;
   justify-content: center;
-  width: 90%;
+  width: 70%;
 }
 
 .logo {
   margin-right: 60%;
   background-repeat: no-repeat;
-  width: 200px;
-  height: 74px;
-  height: 28px;
-  font-family: 'Inter',sans-serif;
+  font-family: 'Inter', sans-serif;
   font-style: normal;
   font-weight: 600;
   font-size: 20px;
   line-height: 28px;
   margin-top: 10px;
-  color: #28352f;
+  margin-left: 5px;
 }
 
-.toolbar-container a{
+.toolbar-container a {
   text-decoration: none;
   font-size: 35px;
   display: block;
+  color: #28352f;
+}
+
+.toolbar-container a:hover {
+  color: #e71010;
 }
 
 .headline {
   display: flex;
   flex-direction: column;
+  width: 375px;
 }
 
 .primary-headline {
-  background-image: url(image/header-illsutration-mobile@2x.png);
+  background-image: url(image/header-illsutration-mobile@2x.JPEG);
```

Comment:
The background image is not loaded on the UI and this is because you are not using the correct name.
Please find here the correct name of your background image: 
`background-image: url(image/header-illsutration-mobile@2x.jpeg);`


![image](https://user-images.githubusercontent.com/83596135/192059187-5e84214b-4505-4e59-ac42-ec73de1b6eb4.png)

- [ ] Please refactor this line of code in order for the background image to be loaded on the UI.